cypress/e2e/AddCartPage.cy.ts

```diff
@@ -0,0 +1,19 @@
+import { AddCartFunctions } from '../pageFunction/AddCartFunctions';
+import {navigate} from '../utils/navigate';
+/* Author: Srajen Singh*/
+beforeEach(()=>{
+    
+    cy.fixture('requiredData').then((data)=>{
+        const url = data.url;
+        navigate(url);
```

Comment:
Url definition should be outside the before each block

Reply:
@ajinkyaces sure I will make changes in next pr